html-forms/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Html Forms</title>
+</head>
+<body>
+  <form action="http://www.example.com">
+    <h1>Coding Form!</h1>
+    <div>
+      Email
+      <br/>
```

Comment:
Hi Devan, please use divs to move form elements onto new lines instead of break statements. brs should only be used to split paragraphs of texts onto new lines